.golangci.yml

```diff
@@ -6,6 +6,7 @@ run:
     - seccomp
     - selinux
     - test
+    - exclude_graphdriver_btrfs
```

Comment:
May I ask you what the reason for this exclusion is? Can we re-add it?

Reply:
For some reason, refreshing the images caused lint errors in containers storage because btrfs wasn't installed on the images (a failed #include) . I should add a comment though

Reply:
I see that issue as well, I will fix this in some time since I'm currently working on a related topic :)